packages/fixie/package.json

```diff
@@ -23,7 +23,7 @@
   "dependencies": {
     "@apollo/client": "^3.8.1",
     "@types/apollo-upload-client": "^17.0.2",
-    "ai-jsx": "workspace:*",
+    "ai-jsx": "^0.18.1",
```

Comment:
@NickHeiner why does this package need to depend on ai-jsx at all?

Reply:
Apparently the answer is for the "PropsOfComponent" type -- I suspect it's not worth bloating the CLI/frontend with the (quite large) dependency graph of ai-jsx, but that's not new with this change :)

Reply:
Yes, it's just for these two imports:

```ts
import { FunctionResponse as FunctionResponseNode, FunctionCall as FunctionCallNode } from 'ai-jsx/core/conversation';
import { PropsOfComponent } from 'ai-jsx';
```

I'm happy to say this isn't worth it and we should just manually copy. It shouldn't be bloating the frontend bundle because it's just types.